app/models/DepartureMessageNotification.scala

```diff
@@ -54,15 +59,18 @@ object DepartureMessageNotification {
     }
 
   def fromRequest(request: DepartureResponseRequest[NodeSeq], timestamp: LocalDateTime): DepartureMessageNotification = {
-    val messageId    = MessageId.fromIndex(request.departure.messages.length)
-    val departureUrl = requestId(request.departure.departureId)
+    val oneHundredKilobytes = 100000
```

Comment:
The whole push message can't be > 100k so we'll only be able to send messages < 99k

Reply:
Cool, I changed the comparison operator to less than rather than less than or equal to